app/views/archived/petitions/show.html.erb

```diff
@@ -72,6 +72,73 @@
     </div>
 
     <div class="about-petitions">
+      <% if debate_outcome = @petition.debate_outcome %>
```

Comment:
Possibly, but we have to deal with both the old archived petitions and the new archived petitions and sometimes taking DRY to the limit isn't advisable.